src/main/java/com/minecolonies/coremod/client/gui/GuiFactory.java

```diff
@@ -0,0 +1,36 @@
+package com.minecolonies.coremod.client.gui;
+
+import java.util.Set;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraftforge.fml.client.IModGuiFactory;
+
+/**
+ * This class is required for Forge to initialize the in-game GUI.
+ */
+public class GuiFactory implements IModGuiFactory 
+{
+    @Override
+    public void initialize(Minecraft minecraftInstance) 
+    { }
+    
+    @Override
+    public Class<? extends GuiScreen> mainConfigGuiClass() 
+    {
+        return ConfigGUI.class;
+    }
+    
+    @Override
+    public Set<RuntimeOptionCategoryElement> runtimeGuiCategories() 
+    {
+        return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1168)